lib/src/callable.dart

```diff
@@ -65,7 +65,12 @@ export 'callable/user_defined.dart';
 ///   access a string's length in code points.
 @sealed
 abstract class Callable extends AsyncCallable {
-  /// Creates a callable with the given [name] and [arguments] that runs
+  @Deprecated('Use `new Callable.function` instead.')
+  factory Callable(String name, String arguments,
+          ext.Value callback(List<ext.Value> arguments)) =>
```

Comment:
It seems like I saw the other function type syntax used elsewhere in this change. Are you mixing them by choice or did habit take over?

Reply:
I usually use new-style when it's necessary to avoid a typedef, but old-style for defining parameters because it's more compact.